Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+    "setup_form_tabs_for_errors" (subject) {
+
+        Admin.log("[core|setup_form_tabs_for_errors] setup form tab's errors", subject);
 
         // Switch to first tab with server side validation errors on page load
-        jQuery('form', subject).each(function() {
-            Admin.show_form_first_tab_with_errors(jQuery(this), '.sonata-ba-field-error');
+        jQuery("form", subject).each(function () {
+
+            Admin.show_form_first_tab_with_errors(jQuery(this), ".sonata-ba-field-error");
+
         });
 
         // Switch to first tab with HTML5 errors on form submit
-        jQuery(subject)
-            .on('click', 'form [type="submit"]', function() {
-                Admin.show_form_first_tab_with_errors(jQuery(this).closest('form'), ':invalid');
-            })
-            .on('keypress', 'form [type="text"]', function(e) {
-                if (13 === e.which) {
-                    Admin.show_form_first_tab_with_errors(jQuery(this), ':invalid');
+        jQuery(subject).
+            on("click", "form [type=\"submit\"]", function () {
+
+                Admin.show_form_first_tab_with_errors(jQuery(this).closest("form"), ":invalid");
+
+            }).
+            on("keypress", "form [type=\"text\"]", function (e) {
+
+                if (e.which === 13) {
+
+                    Admin.show_form_first_tab_with_errors(jQuery(this), ":invalid");
+
                 }
-            })
-        ;
+
+            });
+
     },
 
-    show_form_first_tab_with_errors: function(form, errorSelector) {
-        Admin.log('[core|show_form_first_tab_with_errors] show first tab with errors', form);
+    "show_form_first_tab_with_errors" (form, errorSelector) {
+
+        Admin.log("[core|show_form_first_tab_with_errors] show first tab with errors", form);
 
-        var tabs = form.find('.nav-tabs a'), firstTabWithErrors;
+        let tabs = form.find(".nav-tabs a"),
+            firstTabWithErrors;
 
-        tabs.each(function() {
-            var id = jQuery(this).attr('href'),
+        tabs.each(function () {
+
+            let id = jQuery(this).attr("href"),
                 tab = jQuery(this),
-                icon = tab.find('.has-errors');
+                icon = tab.find(".has-errors");
 
             if (jQuery(id).find(errorSelector).length > 0) {
+
                 // Only show first tab with errors
                 if (!firstTabWithErrors) {
-                    tab.tab('show');
+
+                    tab.tab("show");
                     firstTabWithErrors = tab;
+
                 }
 
-                icon.removeClass('hide');
+                icon.removeClass("hide");
+
             } else {
-                icon.addClass('hide');
+
+                icon.addClass("hide");
+
             }
+
         });
+
     },
 
-    setup_inline_form_errors: function(subject) {
-        Admin.log('[core|setup_inline_form_errors] show first tab with errors', subject);
+    "setup_inline_form_errors" (subject) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'setup_inline_form_errors' should be before 'show_form_first_tab_with_errors'.